src/BaseTemplates/WebAPI/project.json

```diff
@@ -25,12 +25,25 @@
     "dnxcore50": { }
   },
 
-  "exclude": [
+  "tools": {
+    "dotnet-publish-iis": "1.0.0-*"
+  },
+
+  "content": [
     "wwwroot",
+    "appsettings.json"
```

Comment:
OK. Honestly I don't know how you put together the complete templates and what the relation between BaseTemplates and Rules is. If there was no Views or appSetting.json in the folder where the project.json file lives I left it out. 

I will try to figure this out and update the PR again.